flake.nix

```diff
@@ -56,6 +56,7 @@
         ./rust.nix
         ./subwasm.nix
         ./scripts/zombienet/flake-module.nix
+        ./centauri-codegen.nix
```

Comment:
please move out of root folder, into `.nix` or into `inputs/ComposableFi/centauri/flake-module.nix`